Neos.ContentRepository.Core/Classes/Feature/WorkspacePublication/Exception/NoChangesException.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Neos.ContentRepository package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+declare(strict_types=1);
+
+namespace Neos\ContentRepository\Core\Feature\WorkspacePublication\Exception;
+
+use Neos\ContentRepository\Core\SharedModel\Workspace\WorkspaceName;
+
+/**
+ * @api thrown as part of command handling in case
```

Comment:
```suggestion
 * @api thrown as part of command handling in case of empty publish and rebase operations
```

Fixing myself here lazy style